### DISCUSSION TITLE
Not able to render the jupyter notebook in my repository

### DISCUSSION THREAD

**Discussion by saurav-sabu** [community/community#124617] [Repositories]
### Repository

Bug

### Body

![image](https://github.com/community/community/assets/59791440/498563ea-4a00-4080-b75e-cffd87ad9bff)

Hi,

I am not able to render the jupyter notebook in the github. Kindly need help

**Comment by LuluW8071** [comment]
This is case for everyone. It could be due to github using new version of notebook renderer `nbformat` and `nbconvert`.

**Reply by saurav-sabu** [reply]
Thank you🙌🏼

**Comment by CopperEagle** [comment]
Hey @saurav-sabu :wave: 

Yes, seems to affect the whole site. I experience the same and so do these guys #124597. Keep looking at [the status page](https://www.githubstatus.com/) for updates on this. However, it is Sunday so don't expect a fix before tomorrow.

**Reply by saurav-sabu** [reply]
Yes..

**Comment by davevad93** [comment]
Hi @saurav-sabu ,

As I said before in this [discussion](https://github.com/orgs/community/discussions/124597) a workaround would be using [nbviewer](https://nbviewer.org/) or using [ReviewNB](https://www.reviewnb.com/?utm_source=reviewnb_blog) which is available on GitHub [marketplace](https://github.com/marketplace/review-notebook-app/plan/MDIyOk1hcmtldHBsYWNlTGlzdGluZ1BsYW4yMDIz#plan-2023). ReviewNB allows to view `ipynb` files from private repos while with nbviewer you can only view those from public ones.

**Reply by saurav-sabu** [reply]
Thank you! Will try this solution

**Reply by grandmas-favorite** [reply]
Nice one @davevad93! That's what I would have suggested

**Comment by PuspanjaliMuduli17** [comment]
did they resolve the issue ?I am facing the issue till now-_-

**Reply by saurav-sabu** [reply]
Now it is resolved

**Comment by mph4** [comment]
Hi Everyone - very sorry for the inconvenience here. We've fixed the problem and are also implementing improvements that we've learned from this to help prevent it (and better respond to any future issues) going forward.

**Reply by ilama2** [reply]
Hi, it's showing up for me now. What should I do?
